.github/workflows/pr.yml

```diff
@@ -28,3 +28,10 @@ jobs:
       ORG_GHPAGES_DEPLOY_KEY: ${{ secrets.ORG_GHPAGES_DEPLOY_KEY }}
     with:
       publish-folder: pr-${{ github.event.number }}
+
+  playwright:
+    name: Playwright
+    needs: artifacts
```

Comment:
Not a fan of depending/waiting on the build+publish of all these artefacts. We don't really need a publication to test this.
This also means that `tests-playwright.yml` depends on an active deployment, and can't run isolated on a branch's code

Reply:
Regarding the first paragraph I understand your point, see my previous reply.
Regarding the second paragraph is it possible to have branch's code without an active deployment?

Reply:
Yes, by running the `http-server` in the workflow run (previous reply)

Reply:
Changed in new commit.